src/Components/Navbar/style.ts

```diff
@@ -1,12 +1,60 @@
 import styled from "styled-components";
+import title from "../../assets/images/navbar-bg.png";
+import { Link } from "react-router-dom";
 
 export const NavStyle = styled.div`
```

Comment:
```suggestion
export const NavStyle = styled.nav`
```
styles can be applied directly to nav component

Reply:
If I make changes, the navbar background is not being applied